assets/app/scripts/filters/resources.js

```diff
@@ -24,7 +24,8 @@ angular.module('openshiftConsole')
       "encodedDeploymentConfig": ["openshift.io/encoded-deployment-config"],
       "deploymentVersion": ["openshift.io/deployment-config.latest-version"],
       "displayName": ["openshift.io/display-name"],
-      "description": ["openshift.io/description"]
+      "description": ["openshift.io/description"],
+      "buildNumber": ["openshift.io/build.number"]
```

Comment:
i guess we can go with this for now, worst case our Docker Labels and our k8s Annotations will follow completely different patterns.